Chart: update Overview (#7056)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chart/02 Overview/10 Key Features.md

> - **Series Options**    

Reviewer: Maybe `Series Types`

> The Chart component features over 20 [types of series](/concepts/05%20UI%20Components/Chart/11%20Series%20Types/10%20Line%20Series.md '/Documentation/Guide/UI_Components/Chart/Series_Types/'). [Combine](/concepts/05%20UI%20Components/Chart/10%20Series/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Series/Overview/') them, [customize](/Documentation/Guide/UI_Components/Chart/Series/Customize_Appearance/) appearance, and configure individual [points](/concepts/05%20UI%20Components/Chart/14%20Series%20Points/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Series_Points/Overview/').

Reviewer: I think we can consider adding all map of different chart types here (similar with [kendo](https://www.telerik.com/kendo-jquery-ui/charts or [MUI](https://mui.com/x/react-charts/#all-mui-x-charts-components)) 
we can use the style from all component list

Please take a look to the [comparison](https://devexpress.sharepoint.com/:x:/s/devextreme/EXyj0qNp6fxGiSm5vLsrkroBXF_J6j2AKspMlGNzpxp5nw?e=JkhWfv&nav=MTVfezNCMzUxODJDLUYyNzctNDk2OS1CRUI2LUIyODZGOTFDQzVDNH0 
) that we have

- **Informational Elements**    
You can enhance Chart with additional information elements, such as the following:

    - [Legend](/concepts/05%20UI%20Components/Chart/35%20Legend/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Legend/Overview/')
    - [Grid](/concepts/05%20UI%20Components/Chart/55%20Grid/00%20Grid.md '/Documentation/Guide/UI_Components/Chart/Grid/')
    - [Title and Subtitle](/concepts/05%20UI%20Components/Chart/58%20Title%20and%20Subtitle.md '/Documentation/Guide/UI_Components/Chart/Title_and_Subtitle/')
    - [Strips](/concepts/05%20UI%20Components/Chart/60%20Strips.md '/Documentation/Guide/UI_Components/Chart/Strips/')
    - [Constant Lines](/concepts/05%20UI%20Components/Chart/65%20Constant%20Lines.md '/Documentation/Guide/UI_Components/Chart/Constant_Lines/')
    - [Error Bars](/concepts/05%20UI%20Components/Chart/85%20Error%20Bars.md '/Documentation/Guide/UI_Components/Chart/Error_Bars/')

> - **Interactive Features**    

Reviewer: Let's add a drill-down feature https://js.devexpress.com/React/Demos/WidgetsGallery/Demo/Charts/ChartsDrillDown/MaterialBlueLight/

> Add interactivity to your Chart with the following elements:

Reviewer: It would be nice to include `Aggregation` feature in the Overview. We have point and discrete aggregation

    - [Tooltips](/concepts/05%20UI%20Components/Chart/30%20Tooltips/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Tooltips/Overview/')
    - [Crosshair](/concepts/05%20UI%20Components/Chart/80%20Crosshair.md '/Documentation/Guide/UI_Components/Chart/Crosshair/')
    - [Zooming and Panning](/concepts/05%20UI%20Components/Chart/95%20Zooming%20and%20Panning '/Documentation/Guide/UI_Components/Chart/Zooming_and_Panning/')
    - [Client-Side Exporting and Printing](/concepts/05%20UI%20Components/Chart/99%20Client-Side%20Exporting%20and%20Printing '/Documentation/Guide/UI_Components/Chart/Client-Side_Exporting_and_Printing/')

>     - [Adaptive Layout](/concepts/05%20UI%20Components/Chart/89%20Adaptive%20Layout.md '/Documentation/Guide/UI_Components/Chart/Adaptive_Layout/')

Reviewer: Let's add one more group `Customization`
We can mention Multiple Axes, Multiple Panes, Custom Legend Markers, Axis Label Templates, Tooltip templates, [Color Gradients and Patterns](https://js.devexpress.com/React/New/23_1/#Data-Filtering-UI-Customization)
`and more`

- **Series Types**    

> The Chart component features over 20 [types of series](/concepts/05%20UI%20Components/Chart/11%20Series%20Types/10%20Line%20Series.md '/Documentation/Guide/UI_Components/Chart/Series_Types/'). The most popular ones are the following:

Reviewer 2: ```suggestion
The Chart component includes over 20 [series types](/concepts/05%20UI%20Components/Chart/11%20Series%20Types/10%20Line%20Series.md '/Documentation/Guide/UI_Components/Chart/Series_Types/'), with the most popular ones being:
```

    - [Bubble](/Documentation/Guide/UI_Components/Chart/Series_Types/Bubble_Series/)
    - [Financial](/Documentation/Guide/UI_Components/Chart/Series_Types/Financial_Series/)
    - [Range](/Documentation/Guide/UI_Components/Chart/Series_Types/Range_Series/)

    [Combine](/concepts/05%20UI%20Components/Chart/10%20Series/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Series/Overview/') them, [customize](/Documentation/Guide/UI_Components/Chart/Series/Customize_Appearance/) appearance, and configure individual [points](/concepts/05%20UI%20Components/Chart/14%20Series%20Points/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Series_Points/Overview/').

- **Data Sources**    
Chart can load and update data from various [data source](/concepts/70%20Data%20Binding/00%20Specify%20a%20Data%20Source/10%20Local%20Array.md '/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Local_Array/') types. Bind data [directly](/Documentation/Guide/UI_Components/Chart/Data_Binding/Bind_Series_to_Data/#Directly) or use a [series template](/Documentation/Guide/UI_Components/Chart/Data_Binding/Bind_Series_to_Data/#Using_a_Series_Template).

- **Informational Elements**    

> You can enhance Chart with additional information elements, such as the following:

Reviewer 2: ```suggestion
You can enhance the Chart with additional information elements, such as:
```

    - [Client-Side Exporting and Printing](/concepts/05%20UI%20Components/Chart/99%20Client-Side%20Exporting%20and%20Printing '/Documentation/Guide/UI_Components/Chart/Client-Side_Exporting_and_Printing/')
    - [Adaptive Layout](/concepts/05%20UI%20Components/Chart/89%20Adaptive%20Layout.md '/Documentation/Guide/UI_Components/Chart/Adaptive_Layout/')
    - [Drill-Down](/Documentation/Guide/UI_Components/Chart/Drill-Down_Chart/) 

- **Customization**    
The following Chart customizations are available:

    - [Multiple Axes](/Documentation/Guide/UI_Components/Chart/Axes/Multi-Axis_Chart/)
    - [Multiple Panes](/Documentation/Guide/UI_Components/Chart/Panes/Multi-Pane_Chart/)
    - [Custom Legend Markers](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/CustomLegendMarkers/)
    - [Axis Label Templates](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/AxisLabelsTemplates/) 
    - [Tooltip Templates](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/TooltipHTMLSupport/)
    - [Color Gradients and Patterns](/Documentation/Guide/UI_Components/Chart/Series/Customize_Appearance/)

- **Data Aggregation**    

> A Chart with too many series points may slow down performance and be difficult to interpret. To enhance analysis, you can [aggregate](/Documentation/Guide/UI_Components/Chart/Data_Aggregation/) the series points. DevExtreme Chart offers point and discrete aggregation modes.

Reviewer 2: ```suggestion
Charts with too many series points can impact performance and readability. To improve analysis, DevExtreme Chart supports point and discrete [aggregation](/Documentation/Guide/UI_Components/Chart/Data_Aggregation/) modes.
```
